examples/mfa/example.go

```diff
@@ -77,13 +78,9 @@ func main() {
 		fmt.Println("New PK token generated")
 
 		// Verify our pktoken including the cosigner signature
-		verifier := client.PKTokenVerifier{
-			AllowedProviders: []client.OpenIdProvider{provider},
-			AllowedCosigners: []client.CosignerProvider{cosignerProvider},
-		}
-
-		// Verify our pktoken including the cosigner signature
-		if err := verifier.Verify(context.TODO(), pkt); err != nil {
+		providerVerifier := verifier.NewProviderVerifier(provider.Issuer(), provider.CommitmentClaim(), verifier.ProviderVerifierOpts{})
```

Comment:
If I'm reading this correctly, this introduces a vulnerability in that it removes the  `aud=client-id` check.

Reply:
This supports the idea that we should have a GoogleVerifier that requires the `aud=client-id` check.